samples/snippets/src/test/java/com/example/bigquerystorage/WriteToDefaultStreamIT.java

```diff
@@ -73,7 +73,11 @@ public void setUp() {
     // Create a new dataset and table for each test.
     datasetName = "WRITE_STREAM_TEST" + UUID.randomUUID().toString().substring(0, 8);
     tableName = "DEFAULT_STREAM_TEST" + UUID.randomUUID().toString().substring(0, 8);
-    Schema schema = Schema.of(Field.of("test_string", StandardSQLTypeName.STRING));
+    Schema schema = Schema.of(
```

Comment:
Where is the actual test?

Reply:
https://github.com/googleapis/java-bigquerystorage/blob/main/samples/snippets/src/test/java/com/example/bigquerystorage/WriteToDefaultStreamIT.java

has the function testWriteToDefaultStream(). This calls WriteToDefaultStream.writeToDefaultStream() which does the test.